templates/zerver/help/view-the-first-message-in-a-stream.md

```diff
@@ -0,0 +1,19 @@
+# View the first message in a stream
+
+The Zulip [search bar](/help/searching-for-messages), located at the top of
+the [central panel](/help/the-zulip-browser-window), allows you to view the
+first message in a stream using a combination of the `stream` and `near`
+search operators.
+
+1. In the search bar, enter `near:1`. This search operator shows the first
+message in the narrow.
+
+2. Next, add a space to the end of your search query and enter the `stream`
+operator with its appropriate operand to narrow your view to a specific
+stream's messages. For example, if you want to see the first message in the
+stream `foobar`, the appropriate search operator to enter would be
+`stream:foobar`.
```

Comment:
Hm, I don't see an autocomplete for topic names, but I can add a `tip` about further narrowing the view to the topic if you'd like